LICENSE

```diff
@@ -1,7 +1,31 @@
-Node's license follows:
+Node.js is licensed for use as follows:
 
-====
+"""
```

Comment:
see https://github.com/nodejs/node/issues/3979

For now I'm just copying what's already been done—most of which was done without seeking solid legal advice (and some of it done after ignoring some legal advice ..), now the Foundation has resources we should take it there.

Can we agree to merge this and update from `master` down when we have proper legal advice to work with?